ext/OceananigansMakieExt.jl

```diff
@@ -23,7 +23,21 @@ end
 convert_arguments(pl::Type{<:AbstractPlot}, f::Field) =
     convert_arguments(pl, convert_field_argument(f)...)
 
-function flattened_cpu_interior(f)
+"""
+    make_plottable_field(f)
```

Comment:
```suggestion
    make_plottable_array(f)
```

I know I suggested field but reading the docstring it seems that "array" is the right word to use here